cluster.yml

```diff
@@ -174,3 +174,35 @@
     - { role: kubernetes/preinstall, when: "dns_mode != 'none' and resolvconf_mode == 'host_resolvconf'", tags: resolvconf, dns_late: true }
   environment:
     DOCKER_API_VERSION: 1.39
+
+- hosts: all
+  gather_facts: False
+  any_errors_fatal: "{{ any_errors_fatal | default(true) }}"
+  become: true
+  tasks:
+    - name: "lvm2"
+      package:
+        name: lvm2
+        state: present
+
+- hosts: all
+  gather_facts: False
+  any_errors_fatal: "{{ any_errors_fatal | default(true) }}"
+  become: true
+  tasks:
+    - name: "nfs-utils"
```

Comment:
Do we need the nfs utils and commons when we are scaling up? Because currently it is only configured only at cluster creation.